docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -9,6 +9,14 @@
 
     <ol>
         <li>Install <a href="https://nodejs.org/">Node.js</a>, which provides the npm package manager.</li>
+        <li>Install <a href="https://angular.io/">Angular 4</a> and <a href="https://www.typescriptlang.org/">TypeScript</a>:
+            <div>
+                <kbd>npm install @angular/common@latest @angular/compiler@latest @angular/compiler-cli@latest
```

Comment:
No, I just lifted that command from the Angular blog. It sure will be nice to have npm sort out dependencies instead of this!